src/myframe.h

```diff
@@ -498,7 +497,10 @@ struct MyFrame : wxFrame {
                  #else
                  _(L"Toggle &Scaled Presentation View\tF12"));
                  #endif
-        MyAppend(viewmenu, A_ZEN_MODE, _(L"Toggle Zen Mode"));
+        viewmenu->AppendCheckItem(A_SHOWTBAR, _(L"Show Toolbar"));

```

Comment:
why are these moved here? "view" is more about the actual treesheets data view, these are better off in "options" where they were?

Reply:
Ok, I moved them to the options menu then.